### PR TITLE
feat(performance): ultra fast is-odd check algorithm (advanced)

### PR DESCRIPTION
**Pull request by brandon-julio-t**
pls check sir, this is pure raw 50 YOE programming in action sir

proof:

image

timem taken: 00.00s (very fast)

### CODE REVIEW

**Comment by felixgoz** [comment]
Bro, why did you delete all my code? Imagine if your bank uses my code, and when I merge it, a bug causes a bank run, leading to the collapse of the entire world economy. I hope you make wiser decisions next time.

**Comment by brandon-julio-t** [comment]
> Bro, why did you delete all my code? Imagine if your bank uses my code, and when I merge it, a bug causes a bank run, leading to the collapse of the entire world economy. I hope you make wiser decisions next time.

but it’s 00.00s sir

**Comment by cipher416** [comment]
> > Bro, why did you delete all my code? Imagine if your bank uses my code, and when I merge it, a bug causes a bank run, leading to the collapse of the entire world economy. I hope you make wiser decisions next time.
> 
> but it’s 00.00s sir

this is very optimized, i see no issue with this pull request

File: .gitattributes
Comment (cipher416) [inline .gitattributes:1]: please i18n

**Pull request closed without merging**